test/integration_serial_test.go

```diff
@@ -420,12 +420,7 @@ func TestTerragruntOutputJson(t *testing.T) {
 	err := runTerragruntCommand(t, fmt.Sprintf("terragrunt apply --terragrunt-json-log --terragrunt-non-interactive --terragrunt-working-dir %s", testPath), &stdout, &stderr)
 	assert.Error(t, err)
 
-	var output map[string]interface{}
-
-	err = json.Unmarshal(stderr.Bytes(), &output)
-	assert.NoError(t, err)
-
-	assert.Contains(t, output["msg"], "Downloading Terraform configurations from git::https://github.com/gruntwork-io/terragrunt.git?ref=v0.9.9")
+	assert.Contains(t, stderr.String(), "Downloading Terraform configurations from git::https://github.com/gruntwork-io/terragrunt.git?ref=v0.9.9")
```

Comment:
The output is not JSON, it displays multiple JSONs,
```
{...}
{...}
{...}
```
This cannot be parsed. That worked before, because there was only one record of log, and it looked like json, but after adding the logs, it broke. This is a set of json logs of `logrus`, and not the output of terraform, I think this is not critical.

Reply:
Yes, it is by intention in this way to be parsed by other applications like Datadog